### PR TITLE
Feature/registration and login

### PR DESCRIPTION
**Pull request by hirsch88**

### CODE REVIEW

File: src/lib/rest-client.ts [hunk 1]
```diff
@@ -0,0 +1,39 @@
+export type TIdentifier = string | number;
+
+export class RestClient {
+
+  constructor() {
+    // TODO:
```
Comment (keniseli) [inline src/lib/rest-client.ts:6]: What's left to do?

File: src/lib/rest-client.ts [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+  }
+
+}
+
+  // public async findOne<T>(id: TIdentifier): Promise<T> {
```
Comment (keniseli) [inline src/lib/rest-client.ts:11]: I guess we don't need those comments anymore, do we?

**Review by keniseli — APPROVED** [review]

File: src/lib/rest-client.ts (hunk 2 above)
Comment (hirsch88) [reply src/lib/rest-client.ts:11]: uu this file should not be in there :-D

File: src/lib/rest-client.ts (hunk 1 above)
Comment (hirsch88) [reply src/lib/rest-client.ts:6]: uu this file should not be in there :-D

File: src/locals/de.ts [hunk 3]
```diff
@@ -26,17 +30,33 @@ export default {
...
+    success_message: 'Es wurde eine E-Mail an <b>%{email}</b> verschickt. Bitte überprüfen Sie Ihre Mailbox.',
+  },
   register: {
     title: 'Registrierung',
     submit: 'Erstellen',
   },
+  verification: {
+    pending: {
+      title: 'Email Bestätigung',
+      message: `Sie sollten in kürze eine Email von uns erhaltn. In der Email befinden sich ein Bestätiguns-Link,
```
Comment (davidheimgartner) [inline src/locals/de.ts]: There are some errors: kürze/ Kürze, erhalten/erhalten

File: src/locals/de.ts [hunk 4]
```diff
@@ -53,6 +73,13 @@ export default {
...
     login_failed: 'Username oder Passwort ist falsch.',
     login_incomplete: 'Bitte geben Sie Username und Passwort an.',
+    register_successful: 'Sie haben sich erfolgreich registriert',
+    register_failed: 'Sie konnten leider nicht registriert werden.',
+    verification_successful: 'Sie wurden erfolgreich verifiziert. Sie können sich nun anmelden und los tippen! :-)',
+    verification_failed: 'Leider konnte Ihr Account nicht verifiziert werden.',
+    forgot_password_failed: 'Leider kann ihr PW nicht zurückgesetzt werden.',
+    reset_password_successful: 'Password wurde erfolgreich zurückgesetzt',
+    reset_password_failed: 'Leider kann ihr PW nicht zurückgesetzt werden.',
   },
```
Comment (davidheimgartner) [inline src/locals/de.ts]: What ist the difference between forgot_password_failed and reset_password_failed?

**Review by davidheimgartner — APPROVED** [review]

**Pull request merged**